CONTRIBUTING.md

```diff
@@ -41,9 +41,12 @@ $ npm run start
 
 ## Pull Requests
 
-We actively welcome your pull requests.
+We actively welcome your pull requests. 
 
-1. Fork the repo and create your branch from `master`.
+Netlify CMS uses the [Forking Workflow](https://www.atlassian.com/git/tutorials/comparing-workflows#forking-workflow) + [Feature Branches](https://www.atlassian.com/git/tutorials/comparing-workflows#feature-branch-workflow). Additionally, PR's should [rebase on rather than merge](https://www.atlassian.com/git/tutorials/merging-vs-rebasing) master to stay current. 
```

Comment:
Last sentence text should read, "Additionally, PR's should be rebased onto master when opened, and again before merging if necessary.